clang/test/SemaSYCL/attr-add-ir-attributes-function-conflict.cpp

```diff
@@ -0,0 +1,42 @@
+// RUN: %clang_cc1 -fsycl-is-device -fsyntax-only -verify %s
+
+// Tests that add_ir_attributes_function causes a warning when appearing with
```

Comment:
Do we care about sycl-std=2017? From what I can tell, this code will emit this diagnostic when passing sycl-std=2017 as well. Is that expected behavior? Will users be able to use compile time properties in older versions of sycl and if not, should the usage be ignored or diagnosed as I think this code will do now.